source/localizable/tutorial/routes-and-templates.md

```diff
@@ -218,14 +224,12 @@ Let's update our `index.hbs` with some HTML for our home page and our links to t
 
 In addition to providing button-style links in each route of our application, we would like to provide a common banner to display both the title of our application, as well as its main pages.
 
-When you create an Ember application with Ember CLI as we did, it generates a template called `application.hbs`.
+We generated a template called `application.hbs`.
```

Comment:
Please see if there's a way to reconcile with https://github.com/emberjs/guides/pull/1533. I think @toddjordan deleted that whole part.